packages/vue-components/src/Tooltip.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <span
+    :class="trigger === 'click' ? 'trigger-click' : 'trigger'"
+    data-mb-component-type="tooltip"
+    tabindex="0"
+  >
+    <portal v-if="targetEl.id" :to="targetEl.id">
+      <slot name="content"></slot>
+    </portal>
+
+    <b-tooltip
+      :target="targetEl"
```

Comment:
> [BootstrapVue warn]: tooltip - The provided target is no valid HTML element.

Could we `v-if` this before the target el has initialised as well? (change the default targetEl to `undefined`) (or a `isMounted` flag with `$el`)

Same for popovers